phpcs.xml

```diff
@@ -4,4 +4,6 @@
         <exclude name="SlevomatCodingStandard.Classes.UnusedPrivateElements.UnusedMethod"/>
         <exclude name="SlevomatCodingStandard.Exceptions.ReferenceThrowableOnly.ReferencedGeneralException"/>
     </rule>
+    <exclude-pattern>tests/phpunit/TaintedPDO.php</exclude-pattern>
```

Comment:
proč?

Reply:
vid vyssie:

phpcs tam chce pridat typy, a phpstan potom vypisuje ze je to nekompatibilne s nadradenou triedou. teraz to neriesim (subory zostavaju nezmenené), kedze v dalsi fazach refactoringu sa bude musiet zrusit zavislost na PDO.